aws-lambda-java-runtime-interface-client/src/main/jni/build-jni-lib.sh

```diff
@@ -7,10 +7,18 @@ SRC_DIR=$(dirname "$0")
 DST_DIR=${1}
 CURL_VERSION=7.77.0
 
-# compile the native library
-docker build -f "${SRC_DIR}/Dockerfile.glibc" --build-arg CURL_VERSION=${CURL_VERSION} -t lambda-java-jni-lib-glibc "${SRC_DIR}"
-docker run --rm --entrypoint /bin/cat lambda-java-jni-lib-glibc /src/aws-lambda-runtime-interface-client.so > "${DST_DIR}"/classes/aws-lambda-runtime-interface-client.glibc.so
+# compile the native library for x86
+mkdir "${DST_DIR}"/classes/x86
+docker build --platform=linux/amd64 -f "${SRC_DIR}/Dockerfile.glibc" --build-arg CURL_VERSION=${CURL_VERSION} -t lambda-java-jni-lib-glibc-x86 "${SRC_DIR}"
+docker run --rm --entrypoint /bin/cat lambda-java-jni-lib-glibc-x86 /src/aws-lambda-runtime-interface-client.so > "${DST_DIR}"/classes/x86/aws-lambda-runtime-interface-client.glibc.so
 
-docker build -f "${SRC_DIR}/Dockerfile.musl" --build-arg CURL_VERSION=${CURL_VERSION} -t lambda-java-jni-lib-musl "${SRC_DIR}"
-docker run --rm --entrypoint /bin/cat lambda-java-jni-lib-musl /src/aws-lambda-runtime-interface-client.so > "${DST_DIR}"/classes/aws-lambda-runtime-interface-client.musl.so
+docker build --platform=linux/amd64 -f "${SRC_DIR}/Dockerfile.musl" --build-arg CURL_VERSION=${CURL_VERSION} -t lambda-java-jni-lib-musl-x86 "${SRC_DIR}"
+docker run --rm --entrypoint /bin/cat lambda-java-jni-lib-musl-x86 /src/aws-lambda-runtime-interface-client.so > "${DST_DIR}"/classes/x86/aws-lambda-runtime-interface-client.musl.so
 
+# compile the native library for ARM
+mkdir "${DST_DIR}"/classes/arm
+docker build --platform=linux/arm64/v8 -f "${SRC_DIR}/Dockerfile.glibc" --build-arg CURL_VERSION=${CURL_VERSION} -t lambda-java-jni-lib-glibc-arm "${SRC_DIR}"
+docker run --rm --entrypoint /bin/cat lambda-java-jni-lib-glibc-arm /src/aws-lambda-runtime-interface-client.so > "${DST_DIR}"/classes/arm/aws-lambda-runtime-interface-client.glibc.so
+
+docker build --platform=linux/arm64/v8 -f "${SRC_DIR}/Dockerfile.musl" --build-arg CURL_VERSION=${CURL_VERSION} -t lambda-java-jni-lib-musl-arm "${SRC_DIR}"
+docker run --rm --entrypoint /bin/cat lambda-java-jni-lib-musl-arm /src/aws-lambda-runtime-interface-client.so > "${DST_DIR}"/classes/arm/aws-lambda-runtime-interface-client.musl.so
```

Comment:
Room for refactoring and code reuse

Reply:
Sure @carlzogh . Let me refactor the code as you proposed and come back with an updated pull request later this week, maybe tomorrow.